src/poetry/console/commands/check.py

```diff
@@ -1,12 +1,25 @@
 from __future__ import annotations
 
+from cleo.helpers import option
+
 from poetry.console.commands.command import Command
 
 
 class CheckCommand(Command):
     name = "check"
     description = "Checks the validity of the <comment>pyproject.toml</comment> file."
```

Comment:
I think we should change the description to something more general. (Probably the same in the docs.) Checking the lockfile has nothing to do with the validity of pyproject.toml.

Reply:
Changed in e0fab00d9dad3e7f82979b47c475d48814c4c127 to match docs